.github/workflows/commit.yaml

```diff
@@ -80,17 +89,16 @@ jobs:
           path: build/bin
 
   e2e:
-    name: Run e2e tests (${{ matrix.runner.os }}, ${{ matrix.args.extension-language }})
+    name: Run e2e tests (${{ matrix.os }}, ${{ matrix.args.extension-language }})
     needs:
       - bin
-    runs-on: ${{ matrix.runner.os }}
+    runs-on: ${{ matrix.os }}
     timeout-minutes: 90  # instead of 360 by default
     strategy:
       fail-fast: false  # don't fail fast as sometimes failures are operating system specific
       matrix:
-        runner:
-          - os: macos-latest
-          - os: ubuntu-latest
+        go-version: [1.16.3]
```

Comment:
sadly there's no top-level variable support in GH actions so we have to find/replace in this file (we also did before this change)